### PR TITLE
JuPyter notebook link in README.md file broke because of file structure change

### PR DESCRIPTION
**Pull request by Dibyendu-IITKGP**
# Description
Previously the folder structured got updated with Issue https://github.com/pybop-team/PyBOP/issues/437. This causes one of the link in README.md file to break.

Steps to reproduce the behaviour

Open README.md file and try to open the link https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/LG_M50_ECM/1-single-pulse-circuit-model.ipynb.

## Issue reference
Fixes # 508

## Review
Before you mark your PR as ready for review, please ensure that you've considered the following:
- Updated the [CHANGELOG.md](https://github.com/pybop-team/PyBOP/blob/develop/CHANGELOG.md) in reverse chronological order (newest at the top) with a concise description of the changes, including the PR number.
- Noted any breaking changes, including details on how it might impact existing functionality.

## Type of change
- [ ] New Feature: A non-breaking change that adds new functionality.
- [ ] Optimization: A code change that improves performance.
- [ ] Examples: A change to existing or additional examples.
- [x] Bug Fix: A non-breaking change that addresses an issue.
- [ ] Documentation: Updates to documentation or new documentation for new features.
- [ ] Refactoring: Non-functional changes that improve the codebase.
- [ ] Style: Non-functional changes related to code style (formatting, naming, etc).
- [ ] Testing: Additional tests to improve coverage or confirm functionality.
- [ ] Other: (Insert description of change)

# Key checklist:

- [x] No style issues: `$ pre-commit run` (or `$ nox -s pre-commit`) (see [CONTRIBUTING.md](https://github.com/pybop-team/PyBOP/blob/develop/CONTRIBUTING.md#installing-and-using-pre-commit) for how to set this up to run automatically when committing locally, in just two lines of code)
- [x] All unit tests pass: `$ nox -s tests`
- [x] The documentation builds: `$ nox -s doctest`

You can run integration tests, unit tests, and doctests together at once, using `$ nox -s quick`.

## Further checks:
- [ ] Code is well-commented, especially in complex or unclear areas.
- [ ] Added tests that prove my fix is effective or that my feature works.
- [ ] Checked that coverage remains or improves, and added tests if necessary to maintain or increase coverage.

Thank you for contributing to our project! Your efforts help us to deliver great software.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,10 +65,11 @@ These include a wide variety of optimisation problems that require careful consi
 Explore our [example notebooks](https://github.com/pybop-team/PyBOP/blob/develop/examples) for hands-on demonstrations:
 
 - [Gravimetric design optimisation (SPM)](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/spm_electrode_design.ipynb)
-- [GITT fitting of an ECM for an LG M50](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/LG_M50_ECM/1-single-pulse-circuit-model.ipynb)
+- [GITT fitting of an ECM for an LG M50](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/1-single-pulse-circuit-model.ipynb)
 - [Non-linear constrained ECM parameter identification](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/ecm_trust-constr.ipynb)
 - [Optimiser comparison for parameter identification](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/multi_optimiser_identification.ipynb)
 - [Parameter identification for spatial pouch cell model](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/pouch_cell_identification.ipynb)
+- [Estimating ECM parameters from HPPC pulse](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/equivalent_circuit_identification_hppc.ipynb)
```
Comment (BradyPlanden) [inline README.md:72]: I think we should limit the number of Thevenin examples to two in the README, do you think this one is better than line 68 or line 69?

**Review by BradyPlanden — COMMENTED** [review]
Thanks for fixing @Dibyendu-IITKGP - just one comment.

File: README.md (hunk 1 above)
Comment (Dibyendu-IITKGP) [reply README.md:72]: Yes, I think HPPC one is better than LG M50 one (68). Should I then remove 68 from README?

Comment (BradyPlanden) [reply README.md:72]: Yep, sounds good to me.

**Review by BradyPlanden — APPROVED** [review]
LGTM - thanks for the fix @Dibyendu-IITKGP!

**Pull request merged**